pol-core/pol/multi/house.cpp

```diff
@@ -804,27 +789,25 @@ Bscript::BObjectImp* UHouse::scripted_create( const Items::ItemDesc& descriptor,
 
   if ( ~flags & CRMULTI_IGNORE_MULTIS )
   {
-    if ( multis_exist_in( pos.x() + md->minrxyz.x() - 1, pos.y() + md->minrxyz.y() - 5,
-                          pos.x() + md->maxrxyz.x() + 1, pos.y() + md->maxrxyz.y() + 5,
-                          pos.realm() ) )
+    if ( multis_exist_in( pos + md->minrxyz - Core::Vec2d( 1, 5 ),
```

Comment:
a bit tough to understand what the expressions are doing here, a variable could help clarify it